src/languages/es.ts

```diff
@@ -784,6 +784,11 @@ export default {
         removed: 'eliminó',
         transactionPending: 'Transacción pendiente.',
         chooseARate: ({unit}: ReimbursementRateParams) => `Seleccione una tasa de reembolso del espacio de trabajo por ${unit}`,
+        unapprove: 'Desaprobar',
+        unapproveReport: 'Unapprove report',
+        headsUp: 'Heads up!',
+        unapproveWithIntegrationWarning: (accountingIntegration: string) =>
+            `Heads up! This report has already been exported to ${accountingIntegration}. Changes to this report in Expensify may lead to data discrepancies and Expensify Card reconciliation issues. Are you sure you want to unapprove this report?`,
```

Comment:
I haven't learned spanish in a while, but isn't this English? :D

Reply:
Haha this got me too at first. 
This is correct Spanish. In the latest patch to Spanish, they've made it more english like.

Reply:
Translations here: https://expensify.slack.com/archives/C06ML6X0W9L/p1719481979895669?thread_ts=1719271714.420999&cid=C06ML6X0W9L